gd/memory/interface.py

```diff
@@ -494,19 +494,29 @@ def enable_object_limit(self) -> None:
-        self.write_bytes(Buffer[0x90, 0x90, 0x90, 0x90, 0x90, 0x90], 0x1FD756); self.write_bytes(Buffer[0x0F, 0x84, 0xFD, 0xFD, 0xFF, 0xFF], 0x1FD756)
-        self.write_bytes(Buffer[0x90, 0x90, 0x90, 0x90, 0x90, 0x90], 0x1FD79A); self.write_bytes(Buffer[0x0F, 0x84, 0xB9, 0xFD, 0xFF, 0xFF], 0x1FD79A)
-        self.write_bytes(Buffer[0x90, 0x90, 0x90, 0x90, 0x90, 0x90], 0x1FD7AF); self.write_bytes(Buffer[0x0F, 0x85, 0xA4, 0xFD, 0xFF, 0xFF], 0x1FD7AF)
-        self.write_bytes(Buffer[0x90, 0x90, 0x90, 0x90, 0x90], 0x20D3B3); self.write_bytes(Buffer[0xE8, 0x58, 0x04, 0x00, 0x00], 0x20D3B3) # Level kicks
-        self.write_bytes(Buffer[0x90, 0x90], 0x1FF7A2); self.write_bytes(Buffer[0x74, 0x6E], 0x1FF7A2)
-        self.write_bytes(Buffer[0xB0, 0x01], 0x18B2B4); self.write_bytes(Buffer[0x74, 0x6E], 0x18B2B4) # Load failed
-        self.write_bytes(Buffer[0xE9, 0xD7, 0x00, 0x00, 0x00, 0x90], 0x20C4E6); self.write_bytes(Buffer[0x0F, 0x85, 0xD6, 0x00, 0x00, 0x00], 0x20C4E6) # Level reset
-
-
+        self.write_bytes(Buffer[0xEB2E], 0x202AAA) # Speedhack kicks
+        self.write_bytes(Buffer[0xEB], 0x15FC2E) # Editor kick
+        self.write_bytes(Buffer[0xEB, 0x0C], 0x1FD557) # Level completion kicks
+        self.write_bytes(Buffer[0x90, 0x90, 0x90, 0x90, 0x90, 0x90], 0x1FD742)
+        self.write_bytes(Buffer[0x90, 0x90, 0x90, 0x90, 0x90, 0x90], 0x1FD756)
+        self.write_bytes(Buffer[0x90, 0x90, 0x90, 0x90, 0x90, 0x90], 0x1FD79A)
+        self.write_bytes(Buffer[0x90, 0x90, 0x90, 0x90, 0x90, 0x90], 0x1FD7AF)
+        self.write_bytes(Buffer[0x90, 0x90, 0x90, 0x90, 0x90], 0x20D3B3) # Level kicks
+        self.write_bytes(Buffer[0x90, 0x90], 0x1FF7A2) 
+        self.write_bytes(Buffer[0xB0, 0x01], 0x18B2B4) # Load failed
+        self.write_bytes(Buffer[0xE9, 0xD7, 0x00, 0x00, 0x00, 0x90], 0x20C4E6) # Level reset
+    def disable_anticheat(self) -> None:
+        self.write_bytes(Buffer[0x74, 0x2E], 0x202AAA)
+        self.write_bytes(Buffer[0x74], 0x15FC2E)
+        self.write_bytes(Buffer[0x74, 0x0C], 0x1FD557)
+        self.write_bytes(Buffer[0x80, 0xBF, 0xDD, 0x02, 0x00, 0x00, 0x00, 0x0F, 0x85, 0x0A, 0xFE, 0xFF, 0xFF, 0x80, 0xBF, 0x34, 0x05, 0x00, 0x00, 0x00, 0x0F, 0x84, 0xFD, 0xFD, 0xFF, 0xFF], 0x1FD742)
+        self.write_bytes(Buffer[0x0F, 0x84, 0xFD, 0xFD, 0xFF, 0xFF], 0x1FD756)
+        self.write_bytes(Buffer[0x0F, 0x84, 0xB9, 0xFD, 0xFF, 0xFF], 0x1FD79A)
+        self.write_bytes(Buffer[0x0F, 0x85, 0xA4, 0xFD, 0xFF, 0xFF], 0x1FD7AF)
+        self.write_bytes(Buffer[0xE8, 0x58, 0x04, 0x00, 0x00], 0x20D3B3)
+        self.write_bytes(Buffer[0x74, 0x6E], 0x1FF7A2)
+        self.write_bytes(Buffer[0x74, 0x6E], 0x18B2B4)
+        self.write_bytes(Buffer[0x0F, 0x85, 0xD6, 0x00, 0x00, 0x00], 0x20C4E6)
 number_to_resolution = {
```

Comment:
Here you should add **TWO** empty lines between `number_to_resolution` and class definition.